lang/en.json

```diff
@@ -18,6 +18,8 @@
         "Groups": "Campaign Groups",
         "Placements": "Campaign Placements",
         "CampaignIds": "Campaign Ids",
-        "CombinedSourcesMediums": "Campaign Source - Medium"
+        "CombinedSourcesMediums": "Campaign Source - Medium",
+        "DoNotChangeCaseOfUtmParametersTitle": "Retain campaign parameter capitalisation",
+        "DoNotChangeCaseOfUtmParametersDescription": "If enabled, Matomo will retain capitalisation of campaign parameters in order to support compatibility with various third party service providers, such as Google. Turning this setting off will turn all campaign parameters into lower case."
```

Comment:
Asked ChatGPT if that sentence can be improved for better understandability. Here's what it came up with. Feel free to use it, or to keep what you have:
```suggestion
        "DoNotChangeCaseOfUtmParametersDescription": "When enabled, Matomo will keep the original capitalization of campaign parameters to ensure compatibility with third-party services like Google. If disabled, all campaign parameters will be converted to lowercase."
```

Reply:
I asked Gillian and she made some tweaks to your suggestion and I have added the same :+1: